libs/ui-toolkit/arrow-variation/src/lib/arrow-variation.component.ts

```diff
@@ -66,7 +70,12 @@ export class ArrowVariationComponent {
    *
    * @param n number to check
    */
+
   public updateArrow(n: number | '') {
```

Comment:
Inside this method there are three lines checking if `n > 0`. Please refactor this so that these repeated checks are removed.